cmd/application/get_test.go

```diff
@@ -39,6 +39,7 @@ func getRootCmdForTest() *cobra.Command {
 	rootCmd.PersistentFlags().Bool("quiet", false, "Squelch non-essential output")
 	rootCmd.PersistentFlags().Bool("no-color", false, "Disable color")
 	rootCmd.PersistentFlags().String("output", "", "Configure output formatting")
+	rootCmd.PersistentFlags().String("default-headers", "", "Configure addtional headers for gate client requests")
```

Comment:
Could use `PersistentFlags().StringArrayP()` to reduce the amount of processing you have to do in `client.go`.